extensions/mongodb/pom.xml

```diff
@@ -45,6 +45,7 @@
                             <Automatic-Module-Name>com.hazelcast.jet.mongodb</Automatic-Module-Name>
                         </manifestEntries>
                     </archive>
+                    <forceCreation>true</forceCreation>
```

Comment:
How is this related to the "standard suffix"?

Reply:
Ah, right. Not strictly related, we just want to force recreating jar files even if nothing changes in given module; we need to shade and relocate some classes, this helps a bit here.

Reply:
I would rather see it in a separate PR, but let's leave it here in this case. I'll be more strict next time :grin: